src/batch-client/index.ts

```diff
@@ -432,6 +433,28 @@ export class ZimbraBatchClient {
 			content
 		}));
 
+	public enableTwoFactorAuth = (options: EnableTwoFactorAuthInput) =>
+		this.jsonRequest({
+			name: 'EnableTwoFactorAuth',
+			body: {
+				name: {
+					_content: options.name
```

Comment:
destructure all options in function argument itself

Reply:
done